### DISCUSSION TITLE
3D export options

### DISCUSSION THREAD

**Discussion by AghHossein** [ansys/pyaedt#4023] [Q&A] [answered]
In SIwave, in 3D export options, there is an option to "ignore cutouts with area less than ...".
Now, I want to use this option in PYAEDT for the export of edb to q3d. How can I activate it?

**Comment by Samuelopez-ansys** [comment]
@ring630 @svandenb-dev

**Comment by hui-zhou-a** [comment, marked as answer]
@AghHossein 
Create a file named as options.config, and place it in the export folder. Define 3D export options in options.config. Please refer to Siwave Scripting Guide.pdf for keywords. (search ScrSetOptionsFor3DModelExport)

Then you can export 3D Model by below method.
`edb.export_q3d("c:/export_folder")`

**Reply by AghHossein** [reply]
Just to follow up, where can I find the Siwave Scripting Guide.pdf, as my problem is essentially finding the right keyword. Thanks!

**Reply by hui-zhou-a** [reply]
@AghHossein click here
![image](https://github.com/ansys/pyaedt/assets/27995305/a1aa7abc-5eb5-4493-8175-bb7370d074ed)
You can find it on the webpage.
![image](https://github.com/ansys/pyaedt/assets/27995305/e1e96470-26bd-40f6-a82d-07c6a36e1df0)

or you can find it in the installation folder
C:\Program Files\AnsysEM\v241\Win64\Help\SIwave
